### ISSUE TITLE
[ActivityIndicator] Update the a11y label and hint

### ISSUE THREAD

**Issue by jmdetloff** [material-components/material-components-ios#3191]
Currently the a11y label is either 'In progress' '{percentage} Percent Complete' or 'Progress Halted'. These are good strings for the value, the label should be something more like "Progress Indicator"
